src/utils/tagObject.ts

```diff
@@ -0,0 +1,34 @@
+const redactableKeys = ["authorization", "x-authorization", "password", "token"];
```

Comment:
Maybe this could be customizable by the customer?

Reply:
Oh good question - we actually already allow the customer to define obfuscation rules via `datadog.yaml` https://docs.datadoghq.com/tracing/setup_overview/configure_data_security/?tab=datadogyaml#replace-rules-for-tag-filtering

This is just a hardcoded list to prevent non-configured payload capture from indexing basic/common secrets. It's not something we want to automatically index. If a user wants to capture these, they'll have to manually set a tag.

Reply:
oh I see, so maybe we should make a link/note about this config option when we'll release this new env var?

Reply:
Yup! And it'll be in the blog post :)